Makefile

```diff
@@ -32,18 +32,14 @@ generate-env-file: ## Generate the environment file for running the tests inside
 
 .PHONY: prepare-docker-runner-image
 prepare-docker-runner-image: ## Prepare the Docker builder image
-	make -C docker build
+	docker pull `grep "FROM " Dockerfile | cut -d ' ' -f 2` || true
```

Comment:
I know this was in the original code, but I'm just curious what pulling `ruby:2.6-slim` before building the image does / adds?

Reply:
So my assumption is that the dockerfile tells it to be built from ruby:2.6-slim. If you run `docker build` then it will by default use the cached version of the ruby:2.6-slim image and then building on top of it. So what we appear to be doing here is pulling the latest image for ruby:2.6-slim to make sure it's up to date. Essentially, making sure we don't use the cached base image.

Whether it's really worth doing that I'm not sure but I've carried it over for simplicity.